cmd/roy/harvest_wikidata.go

```diff
@@ -42,12 +46,15 @@ func harvestWikidata() error {
 		"Roy (Wikidata): Harvesting definitions from: '%s'",
 		config.WikidataEndpoint(),
 	)
-	sparqlMe := spargo.SPARQLClient{}
-	sparqlMe.ClientInit(config.WikidataEndpoint(), config.WikidataSPARQL())
-	sparqlMe.SetUserAgent(config.UserAgent())
-	res := sparqlMe.SPARQLGo()
+	res, err := spargo.SPARQLWithProv(
+		config.WikidataEndpoint(),
+		config.WikidataSPARQL(),
+		config.WikidataProvenanceField(),
+		historyLen,
+		threads,
+	)
 	path := config.WikidataDefinitionsPath()
-	err = ioutil.WriteFile(path, []byte(res.Human), config.WikidataFileMode())
+	err = ioutil.WriteFile(path, []byte(fmt.Sprintf("%s", res)), config.WikidataFileMode())
```

Comment:
Optimization on the original code. This could have been used to simplify things the first time around.